src/OpenAL/OpenToolkit.OpenAL/ALTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenToolkit.Audio.OpenAL
+{
+    internal class ALTest
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Hello!");
+            var device = ALC.OpenDevice(string.Empty);
```

Comment:
Hmm maybe nullchecks... What happens if you remove all your soundcards for a moment from device manager. OpenAL will fail, as far as i know.